### PR TITLE
Ensure excel exports are in xlsx format

### PR DESCRIPTION
**Pull request by KipSigei**
### Changes / Features implemented
- Ensure excel exports are in `xlsx` format instead of `xls`
### Steps taken to verify this change does what is intended
- Tests
### Side effects of implementing this change
- All excel exports will be in the `xlsx` format

**Before submitting this PR for review, please make sure you have:**

  - [x] Included tests
  - [ ] Updated documentation

Closes #

### CODE REVIEW

File: onadata/apps/api/tests/viewsets/test_xform_viewset.py [hunk 1]
```diff
@@ -3539,6 +3540,32 @@ def test_export_form_data_async(self, async_result):
                 export = Export.objects.get(task_id=task_id)
                 self.assertTrue(export.is_successful)
 
+    @override_settings(CELERY_TASK_ALWAYS_EAGER=True)
+    def test_raises_exception_for_xls_export_formats(self):
```
Comment (DavisRayM) [inline onadata/apps/api/tests/viewsets/test_xform_viewset.py:3544]: Test name is a bit misleading; I don't see an exception raised in this test

File: onadata/apps/viewer/models/export.py [hunk 2]
```diff
@@ -68,6 +68,7 @@ class Export(models.Model):
     """
 
     XLS_EXPORT = "xls"
```
Comment (DavisRayM) [inline onadata/apps/viewer/models/export.py:70]: Should we remove `xls` exports ?

File: onadata/apps/viewer/tasks.py [hunk 3]
```diff
@@ -127,12 +127,10 @@ def _create_export(xform, export_type, options):
 @app.task(track_started=True)
 def create_xls_export(username, id_string, export_id, **options):
```
Comment (DavisRayM) [inline onadata/apps/viewer/tasks.py:128]: Should we rename the function ? Or better yet add a new function and deprecate this one ?

File: onadata/apps/api/tests/viewsets/test_xform_viewset.py (hunk 1 above)
Comment (KipSigei) [reply onadata/apps/api/tests/viewsets/test_xform_viewset.py:3544]: The tests asserts the `ExportTypeError` is raised with .xls formats [here](https://github.com/onaio/onadata/pull/2346/files#diff-a7dc39d301394b01b2cfcd02e9dcfc7b3eadca03097090cd2b3a5bf72fa1cdf6R3554)

**Review by DavisRayM — APPROVED** [review]
LGTM, let's test it out on stage

**Review by DavisRayM — DISMISSED** [review]
Seems like a breaking change to me; We'll need to communicate accordingly

**Review by DavisRayM — APPROVED** [review]

**Pull request merged**